aws/microservices/receipts/http-load-balancer.yml

```diff
@@ -4,10 +4,10 @@ metadata:
   name: tb-http-loadbalancer
   namespace: thingsboard
   annotations:
-    kubernetes.io/ingress.class: alb
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/target-type: ip
 spec:
+  ingressClassName: alb
```

Comment:
I think only this file actually matter:

https://github.com/midokura/thingsboard-ce-k8s/blob/master/helm/thingsboard/templates/ingress.yaml

Reply:
but there is no class configured there. What am I missing?

Reply:
So we should add it

Reply:
then it is using the default one. not related to the PR that tries to remove deprecated usages, right?

Reply:
Ah, ok. The idea of the task was to unify evp api and thingsboard ingress configuration, that is why I got confused.